ra-optimisation/rao-api/src/main/java/com/farao_community/farao/rao_api/RaoParameters.java

```diff
@@ -74,15 +75,33 @@ private void readExtensions(PlatformConfig platformConfig) {
 
     //loop flow parameter section
     static final boolean DEFAULT_RAO_WITH_LOOP_FLOW_LIMITATION = false; //loop flow is for CORE D2CC, default value set to false
+    static final boolean DEFAULT_LOOPFLOW_APPROXIMATION = false;
 
     private boolean raoWithLoopFlowLimitation = DEFAULT_RAO_WITH_LOOP_FLOW_LIMITATION;
 
+    /**
+     *  loopflow approximation means using previous calculated ptdf and net position values to compute loopflow
+     *  ptdf is calculated by a sensitivity analysis, net position is derived from current network
+     *  if "loopflowApproximation" is set to "false", then each loopflow computation do a sensi for ptdf;
+     *  if "loopflowApproximation" is set to "true", loopflow computation tries to use previous saved ptdf and netposition.
+     *  note: Loopflow = reference flow - ptdf * net position
+     */
+    private boolean loopflowApproximation = DEFAULT_LOOPFLOW_APPROXIMATION;
+
```

Comment:
I like it.

Not for this PR, but maybe for later... Do you think if would be relevant to have some kind of "in between" approximation ? which would be : 
* we compute the loopflows 1 time per LinearRao, at the beginning of the LinearRao
* except for this first time, inside the LinearRao, we also use the previously calculated shifts

As the LinearRao is called several time during a search-tree, I would be more accurate than the current approximation, but quicker than the accurate mode.

We will have to find the right tradeoff between accuracy and computation times for the loopflow, having those different modes will enable us to do those testing and advise the users for the best configuration.

Reply:
Sure, we can add another parameter to do a inital ptdf for each LinearRao, this means we need to save the ptdf * netpostion value with variant. 

You can have a check already with the current parameter to test if loopflow value changes a lot between a exact ptdf and an approximated ptdf.